### ISSUE TITLE
config: add lsd group to STAFF_GROUPS

### ISSUE THREAD

**Issue by MirruK** [Datateknologerna-vid-Abo-Akademi/date-website#653]
Either add the lsd group to STAFF_GROUPS in production, or change the is_staff attribute to a field on every member. This way
date-admin or the board can go and manually set someone to have access to the admin panel, this is currently not possible.

**Comment by jaspnas** [comment]
We probably don't want to set this per-member, Maybe we should have a separate admin_access group that gives access to admin, and remove all other groups from STAFF_GROUPS

**Comment by MirruK** [comment]
Yes I like that suggestion, the staff_groups thing is not very flexible in all honesty
